arm-network/2015-06-15/swagger/network.json

```diff
@@ -5670,6 +5670,24 @@
       },
       "description": "AddressSpace contains an array of IP address ranges that can be used by subnets"
     },
+    "BgpSettings": {
```

Comment:
The indentation is awful every where. Please make sure that tabs are replaced by spaces.
If you are using Visual Studio, then for json docs set the tabs to 2 spaces.
Select All and then Ctrl + K + D should format the doc correctly.